package/Makefile.winx86

```diff
@@ -7,19 +7,19 @@ NSIS_OPTS     := /V3
 NSIS_DIR      := $(ROOT_DIR)/package/winx86
 NSIS_SCRIPT   := $(NSIS_DIR)/dronin.nsi
 NSIS_TEMPLATE := $(NSIS_DIR)/dronin.tpl
-NSIS_HEADER   := $(BUILD_DIR)/ground/gcs/dronin.nsh
+NSIS_HEADER   := $(PACKAGE_DIR)/$(GCS_PACKAGE_NAME)/dronin.nsh
 
-standalone: 
+standalone: gcs
```

Comment:
I understand what you are saying, but for me, as someone who is building his own code and not jenkins I prefer to have a make target that does everything for me while I take a cup of coffee (and at least @tracernz from IRC seems to share the idea), even if I have to clean the build first. We can have one target with and other without dependencies making everyone happy though.